crates/sui-graphql-rpc/src/context_data/db_data_provider.rs

```diff
@@ -0,0 +1,145 @@
+// Copyright (c) Mysten Labs, Inc.
```

Comment:
This whole file probably shouldn't be copied from the indexer crate, we should have a shared handle on the DB with these convenience functions.